backend/app/resources.py

```diff
@@ -35,23 +46,25 @@ def get(self):
 
     def post(self):
         args = creating_item_parser.parse_args()
-        item = Repository.create_and_add(Item, args)
-        if not item:
-            return 404
-        db.session.commit()
-        return marshal(item, item_marshaller), 201
+        try:
```

Comment:
Once you implement error handler this `try..except` should no longer be necessary